### PR TITLE
Added rails code for dummy model recommendations

### LINKED ISSUE #26: Rails - Model recommendation system/page
### User Story:

As a user, I would find it useful to be able to get a general idea of model performance before locking in a model for training.

### Persona:

Waleed has little experience with machine learning models, and therefore would like to know which model is best suited for his data in a simple metric before committing the resources to training it.

### Feature:

Look a dataset features and parameters to recommend best models for training.

### Business Value:

Users will have an easily understandable metric by which they can evaluate models pre-training.

### Tasks

For each model developed, output generate dataset features that can be intuitively understood by an end user before training. 

### Acceptance Criteria

Backend python script should output a set of model features as well as a list of model recommendations for all models that can be used to compares the models' possible against others pre-training.

### Acceptance Tests

Run model api class via python and run dataset_evaluation to get metrics.

### PR DESCRIPTION
**Pull request by Mohammad4844**
Features:
- Add model recommendation system to recommend based on dataset metadata
- Added logical flow for `models/new`: select dataset > select target column > select one of the recommended models

Testing:
- Go to `models/new`
- Select a dataset
- Select a target column
- You should see a few suggested models and hyperparams
- Select a model, fill in name and description, and create the model
- You should now be in the `model#show` page, with info on model_type, hyperparams, and labels

Issue #26

### CODE REVIEW

**Review by zsabri91 — APPROVED** [review]
Just reviewed it. It looks good to me
